staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/interface.go

```diff
@@ -18,34 +18,22 @@ package validatingadmissionpolicy
 
 import (
 	"k8s.io/api/admissionregistration/v1alpha1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apiserver/pkg/admission"
-	"k8s.io/apiserver/pkg/admission/plugin/webhook/generic"
+	"k8s.io/apiserver/pkg/admission/plugin/cel"
 )
 
-// Validator defines the func used to validate an object against the validator's rules.
-// It expects the inbound object to already have been converted to the version expected
-// by the underlying CEL code (which is indicated by the match criteria of a policy definition).
-type Validator interface {
-	Validate(versionedAttr *generic.VersionedAttributes, versionedParams runtime.Object) ([]PolicyDecision, error)
-}
+// ValidatingAdmissionPolicyValidator wraps the cel.Validator to assist with ValidatingAdmissionPolicy matching and validation
+type ValidatingAdmissionPolicyValidator interface {
```

Comment:
Since this type is already in a `validatingadmissionpolicy` package, a typical reference from another package would be `validatingadmissionpolicy.ValidatingAdmissionPolicyValidator`, which seems unnecessarily verbose. I'm in favor of just calling the type `validatingadmissionpolicy.Validator`. This seems idiomatic and inline with the rest of the k8s codebase.

Reply:
Yes agreed! Done